agenta-backend/agenta_backend/models/converters.py

```diff
@@ -4,7 +4,7 @@
 from agenta_backend.models.db_models import AppVariantDB, ImageDB, TemplateDB
 from agenta_backend.models.api.api_models import (
     AppVariant,
-    Image,
+    ImageOutput,
```

Comment:
ImageExtended # includes the mongodb image id

Reply:
Thank you.